website-gatsby/src/components/animation-loop-example-page.jsx

```diff
@@ -136,7 +138,7 @@ export default class AnimationLoopExamplePage extends Component {
   }
 
   render() {
-    const {name, panel = true, stats, sourceLink} = this.props;
+    const { exampleConfig: { title , path } = {}, panel = true, stats } = this.props;
```

Comment:
We don't use spaces inside curly braces.

In VS Code,

install `prettier` plugin and update 

go to preferences (workspace or global), and set

```
"typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
"javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
```